modin/core/execution/ray/implementations/cudf_on_ray/partitioning/partition.py

```diff
@@ -225,6 +225,8 @@ def mask(self, row_labels, col_labels):
             The row labels to extract.
         col_labels : list of hashable
             The column labels to extract.
+        order_broken : bool, default: False
+            If the value is True, then some optimizations are not applicable.
```

Comment:
the parameter was added but unused :), however, this method also has some ground to implement it

Reply:
This change will be needed [here](https://github.com/modin-project/modin/pull/4880).